website/docs/how-to/how-to-clone-environment.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: How to clone an environment
+---
+
+:::info availability
+
+Environment cloning is planned to be released in **Unleash 4.18**.
+
+:::
+
+Environment cloning is a tool to enable an admin to duplicate an existing environment, including all feature toggles strategies and their state. See [_Cloning environments_](../user_guide/environments#cloning-environments) for more information.
+
+## Step 1: Navigate to the environments page {#step-1}
+
+Navigate to the _Environments_ page in the admin UI (available at the URL `/environments`). Use the navigation menu item "Configure" and select "Environments".
+
+![The admin UI navigation "Configure" submenu with the Environments item highlighted.](/img/clone-environment-1.png)
+
+## Step 2: Select an environment to clone {#step-2}
+
+Select an environment to clone. On the right side, open the actions submenu and select "Clone".
+
+![The "production" environment actions submenu with the Clone option highlighted.](/img/clone-environment-2.png)
+
+## Step 3: Fill in the clone environment form {#step-3}
+
+Give your new environment a name. The name must be unique and cannot be the same as the original environment. The name is pre-filled with a suggestion, but you can change it to whatever you like.
+
+Select an environment type, and which projects to clone the environment configuration in. You also have the option to keep the users permission for the new environment.
+
+![The clone environment form filled with some example data, and the Clone environment button highlighted at the bottom.](/img/clone-environment-3.png)
+
+## Step 4 (optional): Generate a new API token {#step-4}
+
+On the form, you also have the option to generate an API token right away. If you choose to do so, you will need to select the projects enabled for the new token. Once you submit the form a new server-side SDK (CLIENT) token will be generated for you.
```

Comment:
If this is the same form, I think it should be part of the same step as before (step 3).

It could maybe be something like:
```suggestion
You can optionally generate an API token for the new environment right away. Select which projects the token should have access to, and the token will be generated when you submit the form.
```